embulk-input-postgresql/src/main/java/org/embulk/input/postgresql/getter/ArrayColumnGetter.java

```diff
@@ -72,6 +72,8 @@ private ArrayNode buildJsonArray(Object[] elements)
                     case Types.LONGNVARCHAR:
                         arrayNode.add(jsonNodeFactory.textNode((String) v));
                         break;
+                    default:
```

Comment:
`ArrayColumnGetter` didn't support array of `decimal` type.
This change supports `decimal` array, but decimal values will be converted into string values such as `"[""1.0"",""2.0"",""3.0""]"`.
Could you fix the problem?

Reply:
I could remove this but wouldn't it be better if we use this fallback for whatever type we don't support? I mean, in the case it's a decimal or a numeric type, a string could be a perfect selection to represent its value (because it could be up to 131072 digits before the decimal point; up to 16383 digits after the decimal point: https://www.postgresql.org/docs/current/static/datatype-numeric.html). What do you think?

Reply:
@puppycloud 
I think `BigDecimal` is preferable to `String` for `decimal` type.
How about the following codes?
<pre>
    case Types.DECIMAL:
    case Types.NUMERIC:
        arrayNode.add(jsonNodeFactory.numberNode((BigDecimal)v));
        break;
    default:
        arrayNode.add(jsonNodeFactory.textNode(v.toString()));
        break;
</pre>

Reply:
Yes, that's absolutely better. I updated my unit-test to cover this as well. Thanks.